tests/Makefile

```diff
@@ -1,11 +1,11 @@
 OUT=websocket pubsub
-CFLAGS=-O3 -Wall -Wextra
-LDFLAGS=-levent -lpthread -lrt
+CFLAGS=-O0 -g -Wall -Wextra -I../src/http-parser/
+LDFLAGS=-g -levent -lpthread
```

Comment:
this is useful for local tests, but ultimately I think we'll want either `-O3` back or the same `DEBUG=1` logic as in the `Makefile` at the root level.